src/common/mesh_utils.cpp

```diff
@@ -50,4 +50,108 @@ std::shared_ptr<mfem::ParMesh> buildParallelMesh(const std::string& mesh_file, c
   return par_mesh;
 }
 
+// a transformation from the unit disk/sphere (in L1 norm) to a unit disk/sphere (in L2 norm)
+void squish(mfem::Mesh& mesh)
+{
+  int num_vertices = mesh.GetNV();
+  int dim          = mesh.SpaceDimension();
+
+  mfem::Vector vertices;
+  mesh.GetVertices(vertices);
+  mfem::Vector vertex(dim);
+  for (int i = 0; i < num_vertices; i++) {
+    for (int d = 0; d < dim; d++) {
+      vertex(d) = vertices[d * num_vertices + i];
+    }
+
+    double l1 = vertex.Norml1();
+    double l2 = vertex.Norml2();
+    vertex *= (l2 < 1.0e-6) ? 0.0 : (l1 / l2);
+
+    for (int d = 0; d < dim; d++) {
+      vertices[d * num_vertices + i] = vertex(d);
+    }
+  }
+  mesh.SetVertices(vertices);
+}
+
+std::unique_ptr<mfem::Mesh> DiskMesh(int approx_number_of_elements)
```

Comment:
Are these more of test utilities as opposed to general of general use inside of the main code?

Reply:
I think they are useful for more than just tests

Reply:
Agreed. I can see production uses for these simple shapes in certain applications.